src/components/11-button-groups/button-groups--segmented.njk

```diff
@@ -0,0 +1,64 @@
+<h6>Segmented buttons</h6>
+<ul class="usa-button-group usa-button-group--segmented">
+  <li class="usa-button-group__item">
+    <button class="usa-button">Button</button>
```

Comment:
All button in the example use "button" as their text. Is this okay or do we want to use text the more closely reflects a potentional real-world example?

Reply:
OK as is I think

Reply:
@thisisdano this is an outdated snippet. Here's the current example: 
https://github.com/uswds/uswds/pull/3374/commits/0620bf3cfae19a66b16d421231e9af906034566b

it uses a more contextual display.